requestmanager/asyncloader/unverifiedblockstore/unverifiedblockstore.go

```diff
@@ -30,21 +33,26 @@ func New(storer ipld.Storer) *UnverifiedBlockStore {
 // comes in as part of a traversal.
 func (ubs *UnverifiedBlockStore) AddUnverifiedBlock(lnk ipld.Link, data []byte) {
 	ubs.inMemoryBlocks[lnk] = data
+	ubs.dataSize = ubs.dataSize + uint64(len(data))
+	log.Debugw("added in-memory block", "data-size", ubs.dataSize)
 }
 
 // PruneBlocks removes blocks from the unverified store without committing them,
 // if the passed in function returns true for the given link
 func (ubs *UnverifiedBlockStore) PruneBlocks(shouldPrune func(ipld.Link) bool) {
-	for link := range ubs.inMemoryBlocks {
+	for link,data := range ubs.inMemoryBlocks {
 		if shouldPrune(link) {
 			delete(ubs.inMemoryBlocks, link)
+			ubs.dataSize = ubs.dataSize - uint64(len(data))
 		}
 	}
+	log.Debugw("finished pruning in-memory blocks", "data-size", ubs.dataSize)
```

Comment:
`data-size` is not very descriptive, and it lacks a unit.

```suggestion
	log.Debugw("finished pruning in-memory blocks", "total_queued_bytes", ubs.dataSize)
```